guides/catalog/04 Introduction to Blob Storage.ipynb

```diff
@@ -114,7 +114,12 @@
    },
    "outputs": [],
    "source": [
-    "blob = Blob(name=\"hop_field_info\", tags=[\"examples\"], geometry=field_geom)"
+    "blob = Blob(\n",
+    "    name=\"hop_field_info\",\n",
```

Comment:
wasn't the issue here that the name isn't unique so it'll block other people from making one if it's not deleted? can we add a uuid4() onto this or the user namespace?

Reply:
Ah good call- for users without an org, this would conflict wouldn't it

Reply:
ignore previous comment - the blob here would have namespace of org:user-hash or just user-hash, so it'll be unique to the user

The other example which I removed had a non-default namespace that could potentially conflict if other users at their same org hadn't deleted it before

Reply:
Doh, this isn't the right piece of code for what I was commenting about!